app-v2/src/services/wiseoldman.ts

```diff
@@ -70,11 +70,13 @@ export const getDeltaLeaderboard = cache(
 export const getEfficiencyLeaderboards = cache(
   (
     metric: EfficiencyLeaderboardsFilter["metric"],
+    limit: number,
+    offset: number,
     country?: Country,
     playerType?: PlayerType,
-    playerBuild?: PlayerBuild
+    playerBuild?: PlayerBuild,
```

Comment:
I see that you made limit and offset come first in the list of args because you can't have optional args before required ones.

But in this case, it'd be best do just change this to be:

```
    metric: EfficiencyLeaderboardsFilter["metric"],
    country: Country | undefined,
    playerType: PlayerType | undefined,
    playerBuild: PlayerBuild | undefined,
    limit: number,
    offset: number,
```

This way there's no optional args, everything else looks good!

Reply:
The `country` enum doesn't seem to like the `undefined` value. I have this error in local

```
Invalid enum value for 'country'. You must either supply a valid country code, according to the ISO 3166-1 standard.
```

Any idea?

Reply:
Ok should be good